### PR TITLE
user_comment

### PR DESCRIPTION
**Pull request by cuongncFramgia**

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -9,11 +9,12 @@ def index
   def show
     if current_user.present?
       @new_review = current_user.reviews.build
+      @new_comment = current_user.comments.build
       @user_book = UserBook.find_by user_id: current_user.id, book_id: @book.id
       @user_book = UserBook.new if @user_book == nil
     end
       @reviews = @book.reviews.order(updated_at: :desc)
```
Comment (nguyenhad) [inline app/controllers/books_controller.rb]: convention

**Comment by hieunk58** [comment]
OK

File: app/models/review.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class Review < ActiveRecord::Base
   belongs_to :book
   belongs_to :user
+  has_many :comment
```
Comment (hieunk58) [inline app/models/review.rb]: has_many comments

**Comment by hieunk58** [comment]
OK

**Pull request merged**